### PR TITLE
Update for PHPUnit 9.1

### PR DESCRIPTION
**Pull request by sebastianbergmann**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-vendor
-composer.lock
-phpunit.xml
+/.idea
```
Comment (stof) [inline .gitignore]: That one should not be in the project gitignore, but in some local-only rule (or globally in your system). We don't want to maintain ignore rules for all IDEs that could be used when contributing: https://help.github.com/en/github/using-git/ignoring-files

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +0,0 @@
-language: php
-
-sudo: false
-
-php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
```
Comment (stof) [inline .travis.yml]: Why removing the Travis config file entirely ?

File: README.md [hunk 3]
```diff
@@ -1,68 +1,19 @@
...
-
-```bash
-$> composer install --prefer-dist
+composer require phpspec/prophecy-phpunit
 ```
 
-You can read more about Composer on its [official webpage](http://getcomposer.org).
+If you only need this library during development, for instance to run your project's test suite, then you should add it as a development-time dependency:
 
-## How to use it
```
Comment (stof) [inline README.md]: Removing the doc about how to use it from the readme looks wrong to me.

File: composer.json [hunk 4]
```diff
@@ -11,12 +11,19 @@
             "email": "stof@notk.org"
         }
     ],
-    "require": {
-        "php": ">=5.3.3",
-        "phpspec/prophecy": "~1.3"
+    "config": {
+        "platform": {
+            "php": "7.3.0"
```
Comment (stof) [inline composer.json]: Should be removed

**Comment by stof** [comment]
@sebastianbergmann regarding the request of exposing a trait, maybe we could have a trait defining `prophesize()` and a `verifyProphecyDoubles` method. Then, ProphecyTestCase would only import the trait and override `verifyMockObjects` to add the call to `verifyProphecyDoubles`. This way, it would be easier to add support for Prophecy in places where you need to extend from a different base class (you could wire the `verifyMockObjects` integration without having to reimplement all the logic).

File: README.md (hunk 3 above)
Comment (sebastianbergmann) [reply README.md]: This is what https://github.com/sebastianbergmann/prophecy-phpunit/issues/2 is about.

File: .travis.yml (hunk 2 above)
Comment (sebastianbergmann) [reply .travis.yml]: Do you still use Travis? To me it's dead. Of course, this is your choice.

File: composer.json (hunk 4 above)
Comment (sebastianbergmann) [reply composer.json]: May I ask why?

**Comment by sebastianbergmann** [comment]
To be honest, I want to work as little on this as possible. Feel free to implement this any way you like. I'll close this pull request now, I opened it too soon anyway. Let me know how you want to proceed.

File: composer.json (hunk 4 above)
Comment (stof) [reply composer.json]: when the CI runs on PHP 7.4, I want composer to resolve dependencies based on PHP 7.4 to match what would happen for our PHP 7.4 users.

**Pull request closed without merging**